package.json

```diff
@@ -97,10 +102,9 @@
   "scripts": {
     "codecov": "codecov -f coverage/coverage-final.json",
     "lint": "xo",
-    "pretest": "npm run lint",
```

Comment:
`xo` is choking on the top level await and the `await import()` statement.

@travi if you don't mind I'd like to get rid of `xo` altogether. Some of its complaints are helpful, but it broke the code at at least two places, and I think the friction it adds is not worth the benefit. I want to replace it with `prettier`, without any custom config. 

We have solid tests, they will catch all errors. And prettier can do the formatting without any friction.

Reply:
i'm ok with it. i do still see value in linting beyond prettier, but `xo` is a bit too opinionated even for my tastes. i don't think it is worth coming up with with alternative linting beyond the benefits of prettier right now tho, so i'm good with making the switch to just prettier for now.

i do have one request if you do get `xo` removed. it was the only thing preventing `npm ls` from passing because of some odd webpack dependency. would you mind including `npm ls >/dev/null` as a check in the ci workflow to keep us from getting into a situation where we have peer dependencies that are incomaptible?

Reply:
btw, prettier is surprisingly still on my list of things to get comfortable with, so don't be surprised if i have plenty of questions around its impact